# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/nsICookie.idl
# Commit: d01a1a055fec
# Full Hash: d01a1a055fece12b906b938954f984bd69278ef7
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2025-04-30 20:49:53
# Regressor Bug: 1960824
# File Overlap Count: 1
# Description:
#   Bug 1960824 - Introduce nsICookie::SAMESITE_UNSET, r=edgul,webdriver-reviewers,extension-reviewers,cookie-reviewers,valentin,Sasha,robwu
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D246638
# ==============================================================================

diff -r 307ff391dd02 -r d01a1a055fec netwerk/cookie/nsICookie.idl
--- a/netwerk/cookie/nsICookie.idl	Wed Apr 30 08:53:12 2025 +0000
+++ b/netwerk/cookie/nsICookie.idl	Wed Apr 30 08:59:18 2025 +0000
@@ -31,6 +31,7 @@
     const uint32_t SAMESITE_NONE   = 0;
     const uint32_t SAMESITE_LAX    = 1;
     const uint32_t SAMESITE_STRICT = 2;
+    const uint32_t SAMESITE_UNSET = 256;
 
     /**
      * the name of the cookie